tsdb/db_test.go

```diff
@@ -7468,7 +7468,7 @@ func TestAbortBlockCompactions(t *testing.T) {
 	defer func() {
 		require.NoError(t, db.Close())
 	}()
-	// It should NOT be compactible at the beginning of the test
+	// It should NOT be compactable at the beginning of the test
```

Comment:
"compactible" seems correct: https://www.dictionary.com/browse/compactible

Reply:
I went with "-able" because there were already 30 instances of "compactable" in the project, including 6 in that same file.

Reply:
Is there any point in changing it though, considering it's correct? I personally think it should be left be.